cmd/flags/toggle.go

```diff
@@ -0,0 +1,102 @@
+package flags
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"ldcli/cmd/cliflags"
+	"ldcli/cmd/validators"
+	"ldcli/internal/errors"
+	"ldcli/internal/output"
+	"ldcli/internal/resources"
+)
+
+func NewToggleOnCmd(client resources.Client) *cobra.Command {
```

Comment:
I think we also need to set the usage template for these, in order to parse out the required/optional flags

Reply:
Updated. Help now looks like this
```
Turn a feature flag on

Usage:
  ldcli flags toggle-on [flags]

Required flags:
      --environment string   The environment key
      --flag string          The feature flag key
      --project string       The project key

Optional flags:


Global Flags:
  -h, --help                  Help for this command
      --access-token string   LaunchDarkly access token with write-level access
      --analytics-opt-out     Opt out of analytics tracking
      --base-uri string       LaunchDarkly base URI (default "https://app.launchdarkly.com")
  -o, --output string         Command response output format in either JSON or plain text (default "plaintext")
```